packit_service/models.py

```diff
@@ -131,6 +132,11 @@ class GitProjectModel(Base):
     # Example: https://github.com/packit/hello-world.git
     https_url = Column(String)
     project_url = Column(String)
+    source = Column(String, nullable=False)
```

Comment:
What about calling it `instance_url` like in other places across the code and in OGR?

The thing that is (probably) not consistent is if we add protocol or not.

Reply:
agreed with the rename, "source" can literally mean anything

Reply:
fixed